### PR TITLE
Updated file loaders to be consistent with new preprocessing

### PR DESCRIPTION
**Pull request by jworcestBAH**

### CODE REVIEW

File: dataactvalidator/validation_handlers/validator.py [hunk 1]
```diff
@@ -750,14 +752,7 @@ def validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType):
...
             data = record[str(fieldsToCheck[i])]
-            if(data == None):
-                # Set data to empty string so it can be padded with leading zeros
-                data = ""
-            field = fieldsToCheck[i].lower()
-            # Pad field with leading zeros
-            length = interfaces.validationDb.getColumnLength(field, fileType)
-            data = data.strip().zfill(length)
-            query = query.filter(TASLookup.__dict__[tasFields[i]] == str(data))
+            query = query.filter(TASLookup.__dict__[tasFields[i]] == data)
```
Comment (nmonga91) [inline dataactvalidator/validation_handlers/validator.py]: Why is the padding removed from this block?

**Comment by nmonga91** [comment]
Everything else looks good, once my question is addressed, I will go ahead and merge

File: dataactvalidator/validation_handlers/validator.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactvalidator/validation_handlers/validator.py]: That padding is now done in the preprocessing before the data is put into staging.  When I added it there I did not think to remove it here, but this is now duplicate functionality.

Comment (nmonga91) [inline dataactvalidator/validation_handlers/validator.py]: Got it, thanks for clarifying

**Comment by nmonga91** [comment]
Looks good, merging.

**Pull request merged**